### ISSUE TITLE
Upvote questions

### ISSUE THREAD

**Issue by profvjreddi** [harvard-edge/cs249r_book#546]
We would like to capture feedback from learners on whether a specific question is challenging or good. So, it would be great to have a button to upvote a question.

**Comment by kai4avaya** [comment]
Yes - this can be something we can apply. Though we will need to save this data in a database
